### ISSUE TITLE
Buttons must have discernible text

### ISSUE THREAD

**Issue by Khyati007** [khyatit13/Coding-practice#215]
Source: `<button aria-label="" class="icon-end-empty" disabled="" tabindex="-1" type="button"><calcite-icon class="icon icon--start" aria-hidden="true" icon="pencil" scale="s" calcite-hydrated=""></calcite-icon></button>`
Element Location: [".c-jobDetailPanel__editPanelBtnControl",".icon-end-empty[aria-label=\"\"]"]
failureSummary: Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"Tags: [Ljava.lang.String;@61e94def
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
